src/main/java/com/github/bvale/helloSpring/controller/HomeController.java

```diff
@@ -8,6 +8,7 @@
 
     @RequestMapping("/")
     public String hello(){
-        return "Hello World!";
+        String foo = "bar";
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this useless assignment to local variable "foo". ([squid:S1854](https://sonarcloud.io/organizations/bernardovale-github/rules#rule_key=squid%3AS1854))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.github.bvale%3AhelloSpring&pullRequest=1&issues=AWSuJ7bs6dA5b-W2m-sf&open=AWSuJ7bs6dA5b-W2m-sf)

[ISSUE_KEY]: AWSuJ7bs6dA5b-W2m-sf